packages/template-retail-react-app/app/components/badge-group/index.jsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import React from 'react'
+import PropTypes from 'prop-types'
+import {useStyleConfig, Box, Badge} from '@salesforce/retail-react-app/app/components/shared/ui'
+
+const BadgeGroup = (props) => {
```

Comment:
👍 i like this approach to group badges together!

Reply:
This might be a nit, but as a developer if I see that there is a component called XGroup.. I'd expect that there is a component called X co-located in the folder where the XGroup is. E.g. we have `app/components/badge-group`, I would assume there is a `app/components/badge`. 

In our case there isn't, since the Badge is a chakra component. This could be a argument not to have a group component.. but also it could be an argument to simply add a JSDoc header here to explain that this component groups chakra badge components provided an array of badge data's.